lib/containers/utils.pm

```diff
@@ -56,6 +56,13 @@ sub check_runtime_version {
     return check_version($other, $current, qr/\d{2}(?:\.\d+)/);
 }
 
+sub check_min_runtime_version() {
```

Comment:
I'll choose a function name that explicitly said `podman`. Otherwise, LGTM

Reply:
its on utils, Check the previous function, I like the idea to have a checker for versions no mater what product